sdk/synapse/Azure.Analytics.Synapse.AccessControl/README.md

```diff
@@ -13,15 +13,15 @@ The complete Microsoft Azure SDK can be downloaded from the [Microsoft Azure Dow
 For the best development experience, developers should use the official Microsoft NuGet packages for libraries. NuGet packages are regularly updated with new functionality and hotfixes.
 
 ### Install the package
-Install the Azure Synapse Analytics access control client library for .NET with [NuGet][nuget]:
+Install the Azure Synapse Analytics access control client library for .NET with [NuGet](https://www.nuget.org/):
```

Comment:
Consider having more targeted links for Nuget (where it makes sense) so clicking on them lands at/near the desired package. Either a search query or a direct link to the package page.